nekoyume/Assets/_Scripts/UI/Module/Recipe/RecipeView.cs

```diff
@@ -43,7 +44,29 @@ public void Show(RecipeViewData.Data viewData, ItemSheet.Row itemRow)
                 throw new FailedToLoadResourceException<Sprite>(itemRow.Id.ToString());
             iconImage.overrideSprite = itemSprite;
 
-            normalLevelText.fontSharedMaterial = viewData.LevelTextMaterial;
+            var sheet = Game.Game.instance.TableSheets.ItemRequirementSheet;
+            if (!sheet.TryGetValue(itemRow.Id, out var row))
+            {
+                levelBg.gameObject.SetActive(false);
+            }
+            else
+            {
+                var currentAvatarLevel = States.Instance.CurrentAvatarState.level;
+                if (currentAvatarLevel >= row.Level)
+                {
+                    normalLevelText.text = $"Lv. {row.Level}";
+                    normalLevelText.fontSharedMaterial = viewData.LevelTextMaterial;
```

Comment:
Is this line necessary? `normalLevelText.fontSharedMaterial` only update here so why do not use default material with no update?

Reply:
Yes, because material should be changed based on item's grade.